src/PassKit/PKEnums.cs

```diff
@@ -221,17 +223,8 @@ public enum PKPaymentMethodType : ulong
 		Debit,
 		Credit,
 		Prepaid,
-		Store
-	}
-
-	[Mac (11,0)]
-	[Watch (3,0)]
-	[iOS (9,0)]
-	[Native]
-	public enum PKPaymentSummaryItemType : ulong
```

Comment:
that's a breaking change to remove this enum type

Reply:
ahh oops! thanks for catching that - diffs removed and added back and I totally missed it